modules/calib3d/test/test_cameracalibration.cpp

```diff
@@ -2158,19 +2161,24 @@ TEST(CV_RecoverPoseTest, regression_15341)
             vector<Point2f> points2(_points2);
 
             // Estimation of fundamental matrix using the RANSAC algorithm
-            Mat E, E2, E3, R, t;
+            Mat E, E2, R, t;
```

Comment:
I'm checking the different camera case first as the same camera case modifies the points input.